tck/src/main/resources/tests/getHero/input.graphql

```diff
@@ -1,15 +1,39 @@
 {
-  superHero(name:"Iron Man") {
-    
-    name
-    primaryLocation
-    realName
-    superPowers
-    teamAffiliations {
-      name
-      members{
-        name
-      }
+    IronMan: superHero(name: "Iron Man") {
+        ... on SuperHero {
+            name
+            primaryLocation
+            realName
+            superPowers
+            teamAffiliations {
+                name
+                members {
+                    name
+                }
+            }
+        }
+
+        ... on SuperHeroError {
+            errorCode
+        }
+    }
+
+    IronWoman: superHero(name: "Iron Woman") {
+        ... on SuperHero {
+            name
+            primaryLocation
+            realName
+            superPowers
+            teamAffiliations {
+                name
+                members {
+                    name
+                }
+            }
+        }
+
+        ... on SuperHeroError {
+            errorCode
+        }
     }
```

Comment:
I like these tests, but could you add them into a separate directory (separate test) instead of modifying the existing getHero test?

Reply:
Yes.